src/dashify/logging/dashify_logging.py

```diff
@@ -218,9 +218,7 @@ def _merge_dictionaries(dict_1: Dict[str, List[Any]], dict_2: Dict[str, List[Any
                     merged[key] = merged[key][:measurement_id] + dict_2[key] + merged[key][
```

Comment:
if ´measurement_id´ is ´None´, we can set it to ´len(merged[key])´ and get rid of the ´else´ statement right?

Reply:
I think so, the third addendum in `merged[key] = merged[key][:measurement_id] + dict_2[key] + merged[key][measurement_id + len(dict_2[key]) - 1:]` is a bit confusing to me though. So, is the idea with this line to rewrite the past measurements?